modules/auxiliary/scanner/http/springcloud_traversal2.rb

```diff
@@ -0,0 +1,68 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Spring Cloud Config Server Directory Traversal',
+      'Description' => %q{
+        This module exploits an unauthenticated directory traversal vulnerability
+        which exists in Spring Cloud Config versions 2.2.x prior to 2.2.3 and
+        2.1.x prior to 2.1.9, and older unsupported versions. Spring
+        Cloud Config listens by default on port 8888.
+      },
+      'References'  =>
+        [
+          ['CVE', '2020-5410'],
+          ['URL', 'https://tanzu.vmware.com/security/cve-2020-5410']
+        ],
+      'Author'      =>
+        [
+          'Fei Lu', # Vulnerability discovery
+          'bfpiaoran@qq.com', # Vulnerability discovery
+          'Dhiraj Mishra' # Metasploit module
+        ],
+      'DisclosureDate' => '2020-06-01',
+      'License'        => MSF_LICENSE
+    ))
+
+    register_options(
+      [
+        Opt::RPORT(8888),
```

Comment:
It might be helpful to give the user the option to set the path of the installation.

```suggestion
        Opt::RPORT(8888),
        OptString.new('TARGETURI', [true, "The base path to Spring Cloud Config installation", '/']),
```